app/services/search/postgres/comment.rb

```diff
@@ -92,7 +79,7 @@ def self.search_documents(
         # (see https://github.com/forem/forem/pull/4744#discussion_r345698674
         # and https://github.com/rails/rails/issues/15185#issuecomment-351868335
         # for additional context)
-        user_ids = results.pluck(:user_id)
+        user_ids = results.pluck("comments.user_id")
```

Comment:
My thinking here is before, we may have been grabbing `user_id` from `articles` instead of `comments`. That led to plucking `user_ids` that couldn't be found. So I specified the `comments` table. We'll see if this fixes it. We verified that it's not bad data.

Reply:
I think the test case I ran yesterday can be repeated post-merge to confirm whether this fixes the issue.

https://app.honeybadger.io/projects/66984/faults/78976746 was the related error (user.slice raised NoMethodError when /search/feed_content?class_name=Comment ran, intermittently).